src/helpers.js

```diff
@@ -89,10 +89,12 @@ export async function normalizeConfig(givenConfig: ConfigGiven): Promise<Config>
 
   config.tryKeyboard = !!config.tryKeyboard
 
-  if (config.tryKeyboard === true && typeof config.onKeyboardInteractive !== 'function') {
-    config.onKeyboardInteractive = (name, instructions, instructionsLang, prompts, finish) => {
-      if (prompts.length > 0 && prompts[0].prompt.toLowerCase().includes('password')) {
-        finish([config.password])
+  if (config.tryKeyboard === true) {
```

Comment:
I think we could word the check differently to make it
```
if (typeof config.onKeyboardInteractive !== 'function' || config.tryKeyboard)
```

That should satisfy the cases we want